theano/gof/vm.py

```diff
@@ -330,7 +330,7 @@ def __init__(self, nodes, thunks, pre_call_clear,
 
         self.allow_gc = allow_gc
         self.message = ""
-        self.base_apply_stack = [o.owner for o in fgraph.outputs if o.owner]
+        self.base_apply_stack = [o.owner for o in fgraph.outputs if o.owner] # sygi: why it can be False?
```

Comment:
This would happen if one of the output is a Constant. This could happen due to Theano optimization.